bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,2078 @@
+	</channel-type>
+
+	<channel-type id="mainsOnDelay" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Mains on Delay</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="switchingCycleLockOff" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Switching Cycle Lock Off</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="switchingCycleLockOn" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Switching Cycle Lock On</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="compressorIdleTime" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Compressor Idle Time</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="heatingControllerMoreTime" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Heating Controller More Ttime</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="heatingControllerLessTime" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Heating Controller Less Time</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="thermalDisinfectionRunningTime" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Runtime Thermal Disinfection</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="timeHotWaterLock" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Hot Water Lock</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="bivalenceStage" advanced="true">
+		<item-type>Number</item-type>
+		<label>Bivalence Stage</label>
+		<state readOnly="true">
+			<options>
+				<option value="1">One Compressor may run</option>
+				<option value="2">Two Compressors may run</option>
+				<option value="3">Additional Heat Generator may run</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="operatingStatus">
+		<item-type>Number</item-type>
+		<label>Operating Status</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">Heating</option>
+				<option value="1">Hot water</option>
+				<option value="2">Swimming pool / Photovoltaics</option>
+				<option value="3">Utility lock</option>
+				<option value="4">Defrost</option>
+				<option value="5">No requirement</option>
+				<option value="6">Heating ext. energy source</option>
+				<option value="7">Cooling mode</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="errorTime0">
+		<item-type>DateTime</item-type>
+		<label>Timestamp Error 0 in Memory</label>
```

Comment:
```suggestion
		<label>Timestamp Error 0</label>
```

Reply:
There are other channels for the values in `Memory 2`, but guess it's enough to mark those channels and remove the memory in this case :+1:

Reply:
You could also group channels, if that helps.